src/widgets/APITree/APITree.jsx

```diff
@@ -38,7 +39,10 @@ function APITree() {
   const [expanded, setExpanded] = useState([]);
   const [errors] = useEvent("DIAGNOSTICS_COMPLETED", []);
   const [state, dispatch] = useContext();
+
   const api = state.get("nucleoid.api");
+  //eslint-disable-next-line
+  const [apiExists, setApiExists] = useState(Boolean(Object.keys(api).length));
```

Comment:
`api` should be array, assumed it is there, but could be empty so `useState(Boolean(api.length))`